api/src/main/java/org/openmrs/module/dhisreporting/DHISReportingConstants.java

```diff
@@ -85,4 +85,6 @@
 	public static String REPORT_UUID_ONART = "7e162556-ff30-11e6-bc64-92361f002671";
 
 	public static String REPORT_UUID_OTHER = "7e162628-ff30-11e6-bc64-92361f002671";
+	
+	public static String DXFTOADXSWITCH_CONCEPTID = "dhisreporting.config.dxfToAdxSwitch";
```

Comment:
we can rename this from being a concept id to sound like a setting, an example at: https://github.com/jembi/openmrs-module-dhisconnector/blob/master/api/src/main/java/org/openmrs/module/dhisconnector/Configurations.java#L8